lib/spack/llnl/util/tty/__init__.py

```diff
@@ -156,6 +162,10 @@ def msg(message, *args, **kwargs):
 
 
 def info(message, *args, **kwargs):
```

Comment:
We should really combine `tty.msg` and `tty.info`. Does anyone know why they both exist?